File: generated-sources/php-slim/mojang-api/lib/com/github/asyncmc/mojang/api/php/slim/model/ChangeSkinRequest.php
<?php
/**
 * ChangeSkinRequest
 */
namespace com.github.asyncmc.mojang.api.php.slim.server\com.github.asyncmc.mojang.api.php.slim.model;

/**
 * ChangeSkinRequest
 */
class ChangeSkinRequest
{
    
    /** @var \com.github.asyncmc.mojang.api.php.slim.server\com.github.asyncmc.mojang.api.php.slim.model\SkinModel $model */
    private $model;
    
    /** @var string $url The URL which Mojang servers will download and apply the skin*/
    private $url;
}

File: generated-sources/scalaz/mojang-api/src/main/scala/com/github/asyncmc/mojang/api/scalaz/model/ChangeSkinRequest.scala
package com.github.asyncmc.mojang.api.scalaz.model

import argonaut._
import argonaut.EncodeJson._
import argonaut.DecodeJson._

import org.http4s.{EntityDecoder, EntityEncoder}
import org.http4s.argonaut._
import org.joda.time.DateTime
import ChangeSkinRequest._

case class ChangeSkinRequest (
  model: Option[SkinModel],
/* The URL which Mojang servers will download and apply the skin */
  url: String)

object ChangeSkinRequest {
  import DateTimeCodecs._

  implicit val ChangeSkinRequestCodecJson: CodecJson[ChangeSkinRequest] = CodecJson.derive[ChangeSkinRequest]
  implicit val ChangeSkinRequestDecoder: EntityDecoder[ChangeSkinRequest] = jsonOf[ChangeSkinRequest]
  implicit val ChangeSkinRequestEncoder: EntityEncoder[ChangeSkinRequest] = jsonEncoderOf[ChangeSkinRequest]
}
